virtual-organizations/IceCube.yaml

```diff
@@ -58,13 +58,24 @@ DataFederations:
         AllowedOrigins:
           - IceCube_StashCache_origin
           
-      - Path: /icecube/NRP-UNL/PROTECTED
+      - Path: /nrpnb/protected/icecube/
```

Comment:
The custom is to have the top-level component of the origin reflect the VO.  It appears this is indicating the origin.

Can you do:

```
/icecube/nrpnb/protected
```

instead?

Note XRootD has various mechanisms to add or subtract prefixes before it tries to open the file on disk.

Reply:
sure, solved!